codec-http2/src/main/java/io/netty/handler/codec/http2/hpack/Decoder.java

```diff
@@ -0,0 +1,568 @@
+/*
+ * Copyright 2015 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
```

Comment:
Yep. We can't modify the old copyright header. We can only add. I guess it sometimes becomes a good reason to write something from scratch instead of forking when the copyright header becomes longer than the actual code. In an ideal world, people would release their code to the public domain to solve this problem. :-D